examples/interop/swiftui-in-compose/iosApp/iosApp/ComposeViewControllerToSwiftUI.swift

```diff
@@ -0,0 +1,39 @@
+import UIKit
+import SwiftUI
+import shared
+import MapKit
+
+struct ComposeViewControllerToSwiftUI: UIViewControllerRepresentable {
```

Comment:
Maybe just name it as `ComposeViewController`? Otherwise it sounds as we convert Compose to SwiftUI, or embed Compose into SwiftUI

Reply:
We really converting Compose to SwiftUI view in this structure. It need to display Compose in iOS Application.

Reply:
Yeah, but because the example is about embedding swiftui in compose, it is confusing why we tell in the name how to embed Compose into SwiftUI. We can just use another neutral name (or the name from the template)

Reply:
I'd call it `ComposeViewControllerRepresentable`, tbh

Reply:
Good suggestion, Done!